cmd/crossplane/main.go

```diff
@@ -33,12 +35,15 @@ import (
 	"github.com/crossplane/crossplane/apis"
 	"github.com/crossplane/crossplane/cmd/crossplane/core"
 	"github.com/crossplane/crossplane/cmd/crossplane/rbac"
+	"github.com/crossplane/crossplane/internal/version"
 )
 
 type debugFlag bool
+type versionFlag string
```

Comment:
declaring versionFlag as a string is (AFAICT) telling Kong to expect a string value to be passed in along with the `-v`.   Note the `-v, --version=VERSION-FLAG` in the help output.

Just running `crossplane -v` results in the help output and an error message of `crossplane: error: --version: expected string value but got "EOL" (<EOL>)`.

Passing a dummy value for `-v` results in correct operation (the crossplane binary version is printed), giving further evidence that is expecting a string to be passed as the option's value:
```
❯ _output/bin/darwin_arm64/crossplane --v=foo
v1.10.0-rc.0.92.ge7c31b08
```

Instead of a string, this should be a flag/switch - see if you can research how Kong treats those types of command line options 🙏 

full output:
```
/dev/go/src/github.com/crossplane/crossplane @e7c31b08
❯ _output/bin/darwin_arm64/crossplane -v
Usage: crossplane <command>

An open source multicloud control plane.

Flags:
  -h, --help                    Show context-sensitive help.
  -d, --debug                   Print verbose logging statements.
  -v, --version=VERSION-FLAG    Print version and quit.

Commands:
  core start
    Start Crossplane controllers.

  core init
    Make cluster ready for Crossplane controllers.

  rbac start
    Start Crossplane RBAC controllers.

  rbac init
    Initialize RBAC Manager.

Run "crossplane <command> --help" for more information on a command.

crossplane: error: --version: expected string value but got "EOL" (<EOL>)
```